### PR TITLE
Audience in token can be a list too #27

### PR DESCRIPTION
**Pull request by simon349587**
That is supposed to solve https://github.com/HarryMWinters/fastapi-oidc/issues/27, the issue that the `aud` field in a token can be a list as well. OpenID Connect spec regarding that is linked in issue 27.

### CODE REVIEW

**Comment by HarryMWinters** [comment]
Hey @simon349587 

Thanks for the PR. It looks like it might be breaking older versions of Python but I'm happy to drop support for those older versions. Feel free to do so yourself in this branch or I can get to it late this week.

**Comment by HarryMWinters** [comment]
Hey @simon349587! I think we may be ommiting the updated pyproject from this

File: fastapi_oidc/types.py [hunk 1]
```diff
@@ -29,15 +29,14 @@ class IDToken(BaseModel):
 
     """
 
+    model_config = ConfigDict(extra="allow")
+
     iss: str
-    sub: str
+    sub: str | List[str]
```
Comment (HarryMWinters) [inline fastapi_oidc/types.py:35]: This is for the `sub` field. Did you mean to add it to `aud`?

**Comment by simon349587** [comment]
Fixed the small issue and added a test with a list of audiences. Feel free to grab the code, taking credit for it is not important for me.

**Comment by HarryMWinters** [comment]
I updated master and fixed the CI. If you rebase and pull you should be able to add the `| list[str]` and test case with CI passing.

**Comment by simon349587** [comment]
I rebased using your changes in `poetry.lock`.

**Comment by HarryMWinters** [comment]
I went ahead and made the changes so we could release v0.0.10 with the audience field fix. Sorry to move forward without your commits. [Here](https://github.com/HarryMWinters/fastapi-oidc/pull/43)'s what I was hoping for post-rebase.

In any case, v0.0.10 should have the fix and is deployed to pypi already. Please don't hesitate contribute again.

**Comment by simon349587** [comment]
Thanks for taking care of it!

**Pull request closed without merging**